includes/Traits/Register_Routes.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Trait Register_Routes
+ *
+ * @package   Google\Web_Stories\Traits
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Traits;
+
+/**
+ * Trait Register_Routes
+ *
+ * @package Google\Web_Stories\Traits
+ */
+trait Register_Routes {
```

Comment:
Not really sold on this trait.

I don't think it adds enough benefit. It's also confusing and error prone because it requires the consuming class to have a `register_routes` method.

Let's just duplicate the code in the two classes this trait is used.

Reply:
It was used in 6 places. But it is a bit of a pointless trait. I have removed it https://github.com/google/web-stories-wp/pull/7225/commits/f25685c8601e82189f4b324abd659b06fcf32b9a